integrations/shuttle-axum/src/lib.rs

```diff
@@ -0,0 +1,41 @@
+//! Shuttle service integration for the Axum web framework.
+//! ## Example
+//! ```rust,no_run
+//! use shuttle_axum::AxumService;
+//!
+//! async fn hello_world() -> &'static str {
+//!     "Hello, world!"
+//! }
+//!
+//! #[shuttle_axum::main]
```

Comment:
```suggestion
//! #[shuttle_runtime::main]
```
I might be wrong about this :see_no_evil:

Reply:
Either could work, but I think this is better. The dependency on shuttle_runtime is actually only needed for the codegen, but the user doesn't know that, so this makes it clear that it's not a redundant dependency.